code/modules/mob/living/silicon/robot/component.dm

```diff
@@ -77,18 +77,16 @@
 	name = "armour plating"
 	external_type = /obj/item/robot_parts/robot_component/armour
 	energy_consumption = 0
-	max_damage = 60
+	max_damage = 110
 
 /datum/robot_component/actuator
 	name = "actuator"
 	external_type = /obj/item/robot_parts/robot_component/actuator
 	energy_consumption = 0 // seeing as we can move without any charge...
-	max_damage = 50
 
 /datum/robot_component/cell
 	name = "power cell"
-	max_damage = 50
```

Comment:
Unatomic change, made cell's health drop from 50 to 15

Reply:
intentional